Web_Development/exercise-05/ex5.js

```diff
@@ -0,0 +1,162 @@
+const pokemon = document.getElementById("pokemon-display");
+const showMore = document.getElementById("show_More__pokemon");
+const showMoreDetails = document.getElementById("fect_more_details");
+const pokemonUrl = "https://pokeapi.co/api/v2/pokemon";
+
+var nextURl = null;
+var URL_info;
+var searchArray = [];
+
+const fetchPokemonDetails = async (pokemonurl) => {
+  await fetch(pokemonurl)
+    .then((response) => response.json())
+    .then((data = {}) => {
+      let { results, next } = data;
+      URL_info = results;
+      nextURl = next;
+      results.forEach((pokemonData) => {
+        fetchPokemonUrlDetails(pokemonData);
+      });
+    });
+};
+
+showMore.addEventListener("click", async () => {
+  await fetchPokemonDetails(nextURl);
+});
+
+const fetchPokemonUrlDetails = async (data) => {
+  let { url } = data;
+  await fetch(url)
+    .then((response) => response.json())
+    .then((urlData) => {
+      renderPokemonData(urlData);
+    });
+};
+
+const renderPokemonData = (urlData) => {
+  let { name, id, weight, base_experience } = urlData;
```

Comment:
`const` can be used here since we're not reassigning the variables, look for other places where this may be so.